<ui version="4.0" >
 <class>QRL_ParametersManager</class>
 <widget class="QDialog" name="QRL_ParametersManager" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Parameters Manager</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="qrtailab.qrc" >
    <normaloff>:/icons/parameters_icon.xpm</normaloff>:/icons/parameters_icon.xpm</iconset>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="3" row="0" column="0" >
    <widget class="QListWidget" name="blockListWidget" >
     <property name="baseSize" >
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Block Parameters</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QTableWidget" name="parameterTableWidget" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QCheckBox" name="batchCheckBox" >
     <property name="text" >
      <string>batch mode</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="downloadPushButton" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>download</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="uploadPushButton" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="text" >
      <string>upload</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="2" column="1" colspan="3" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="helpButton" >
       <property name="text" >
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>Close Manager</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="hideCheckBox" >
     <property name="text" >
      <string>Hide unnamed Blocks</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>blockListWidget</zorder>
  <zorder>tabWidget</zorder>
  <zorder>batchCheckBox</zorder>
  <zorder>downloadPushButton</zorder>
  <zorder>uploadPushButton</zorder>
  <zorder>hideCheckBox</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>helpButton</zorder>
  <zorder></zorder>
  <zorder>closeButton</zorder>
 </widget>
 <resources>
  <include location="qrtailab.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>QRL_ParametersManager</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>401</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel" >
     <x>464</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
